tests/tck/conftest.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2020 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+import pytest
+
+from nebula2.common.ttypes import DataSet, Row
+from nebula2.data.DataObject import DataSetWrapper
+from tests.tck.utils.nbv import parse
+
+
+def _table(text):
+    lines = text.splitlines()
+    assert len(lines) >= 1
+
+    def parse_line(line):
+        return list(map(lambda x: x.strip(),
+                        filter(lambda x: x, line.split('|'))))
```

Comment:
This will filter out the empty cells, right? Is that as expected?

Reply:
yeah. We don't allow following case `| nothing here |` in expected table, so is safe to filter the empty cells.